src/videojs.hlsjs.js

```diff
@@ -150,8 +150,12 @@ var HlsSourceHandler = {
 if (Hls.isSupported()) {
   var videojs = require('video.js'); // resolved UMD-wise through webpack
 
+  // we could just use `import videojs from 'video.js'`
+  // but that will require a `babel-loader` or similar dependency
+  videojs = videojs.default || videojs;
+
   if (videojs) {
-    videojs.getComponent('Html5').registerSourceHandler(HlsSourceHandler, 0);
+    videojs.getTech('Html5').registerSourceHandler(HlsSourceHandler, 0);
```

Comment:
lets fallback to `getComponent` since its needed for older versions of videojs5
```js
    var html5Tech = videojs.getTech && videojs.getTech('Html5'); // videojs6 (partially on videojs5 too)
    html5Tech = html5Tech || (videojs.getComponent && videojs.getComponent('Html5')); // videojs5
    if (html5Tech) {
      html5Tech.registerSourceHandler(HlsSourceHandler, 0);
    }
```